user-guide/docs/usecases/README.md

```diff
@@ -133,21 +156,37 @@
     | - |
     | ![PR changes](docs/images/20-pr-file-changes.png) |
 
-<details><summary>to Preview Deployment — ⚠️ Feature Not Available Yet</summary>
+    <a id="test-changes">
 
-11. After a minute or so a preview deployment of your use case will be available on the pull request page. Select the preview link generated by Netlify to view your changes similar to how it would be rendered in the final version. If you want to make some tweaks. Visit *your* GitHub repo of the DS Use Case and make changes. As long as the current PR remains open GitHub will automatically pull your changes.
+11. You can [**test your changes** on your local machine](../../../README#testing) using a command prompt (very few commands).
 
-    | link to preview of deployment |
-    | - |
-    | ⚠️ Feature unavailable. Until feature is available, you may view an [outdated image](docs/images/21-pr.png). |
-    <!-- ![PR](docs/images/21-pr.png) | -->
+    > **Note:**
+    > GitHub preview is an inaccurate representation of what will appear on the site.
+    >
+    > - Markdown rendering is less forgiving than [Github's](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) e.g.
+    >     - Use 4 spaces to indent.
+    >     - Add a new line before starting a list.
+    > - Image paths must be:
+    >     - **either** absolute starting at `/user-guide/` ([example](https://github.com/DesignSafe-CI/DS-User-Guide/pull/35/commits/956fc03))
+    >     - **or** relative to page that includes document that loads image
```

Comment:
Remove this, because setting image paths intuitively works on GitHub and MkDocs (on direct file views and included files). I don't know what caused my confusion that I felt I needed to add this.
 
```suggestion
```